src/frontend/components/Felleskomponenter/BlokkerTilbakeKnappModal/BlokkerTilbakeKnappModal.tsx

```diff
@@ -35,24 +28,26 @@ const BlokkerTilbakeKnappModal = () => {
             open={visBlokkerTilbakeKnappModal}
             aria-label={formatMessage({ id: 'felles.blokkerTilbakeKnapp.modal.tittel' })}
```

Comment:
Tror egentlig vi kan fjerne aria-label på modal, for i følge aksel så blir denne automatisk hentet fra Header når header er til stede (som den er her): https://aksel.nav.no/komponenter/core/modal
![image](https://github.com/navikt/familie-ba-soknad/assets/25459913/a94009df-f7b6-4816-aea3-4fe01ed7b8ac)

Reply:
Jeg misforsto det her, så den kommer bare automatisk hvis man bruker heading-prop i Modal og ikke hvis man bruker Modal.Header. Må derfor settes manuelt likevel. Kan ikke bruke header-prop fordi vi er avhengig av SpråkTekst-komponenten (og i header-prop kan du bare sende med plain tekst). Kan evt bytte fra aria-label til aria-labelledby og peke den til headeren selv? Noe ala på bildet under her:
![image](https://github.com/navikt/familie-ba-soknad/assets/25459913/de28fea6-4069-46ec-979b-4183135e9597)

Reply:
Vi kan bruke header-prop om det er best! Er noe som heter formattedMessage som gjøre om teksten til en string og ikke komponent 😊

Reply:
Testet det i går kveld, men problemet er at jeg ikke får overskriften til å bli like stor som vi har den. Kun mulig å velge mellom small og medium, og vi har noe som tilsvarer large eller xl 😬

Reply:
Sånn vi har det nå:
![image](https://github.com/navikt/familie-ba-soknad/assets/25459913/9c8efc70-374f-43ec-97a6-42544beccf35)

Med bruk av heading-prop:
![image](https://github.com/navikt/familie-ba-soknad/assets/25459913/2f442c27-6db3-4161-9681-a22c51d00901)

Reply:
Jeg tenker det er fint at den blir litt mindre hvis det lar oss bruke header propen 🙌